File: example/vwo/net/sample-VWO.cs
var visitor = Ff.NewVisitor("<VISITOR_ID>")
  .IsAuthenticated(true)
  .HasConsented(true)
  .WithContext(new Dictionary<string, object> {
    ["isVIP"] = true,
    ["country"] = "NL",
    ["loginProvider"] = "Google"
    })
  .Build();

  await visitor.FetchFlags();

var flag = visitor.GetFeatureVariableValue(campaignKey, "VWO-flag-cs", "user_123", options);

File: cmd/analyze/competitor-example/vwo/js/sample-VWO.ts
FeatureFlag.start("your_env_id", "your_api_key");

const visitor = FeatureFlag.newVisitor({
    visitorId: "your_visitor_id",
    context: { isVip: true },
});

visitor.on("ready",  (error) => {
    if (error) {
        return;
    }

    const btnColorFlag: string = client.getFeatureVariableValue(campaignKey, 'VWO-flag-js', 'user_123', options);

});

visitor.fetchFlags();